docs/faq/migration/token-migration.md

```diff
@@ -68,12 +69,24 @@ If you migrate yourself but don't have a browser-based wallet, e.g., holding you
 
 You can check your balance using the [LUKSO Mainnet Block Explorer](https://explorer.execution.testnet.lukso.network/) and searching for your address.
 
-You can also add the network to your wallet. This can be done from the migration dApp or by manually adding the RPC endpoint of one of the supported service providers like [GatewayFM](https://gateway.fm/lukso). Have a look at the [Network Properties](../../networks/mainnet/parameters.md) to see the connection details.
+If you want to see the balance within your wallet, you can add LUKSO to your wallet by manually adding the RPC endpoint as described within our official [Network Properties](../../networks/mainnet/parameters.md).
+
+## How to store LYX?
+
+You can store LYX using the [Universal Profile Browser Extension](/install-up-browser-extension) or any Ethereum-compatible wallet or hardware wallet. Further details can be found on the [Wallet Support Page](../lukso/wallet-support.md).
 
-## How to store LYX or LYXe?
+:::caution
 
-You can store both LYX and LYXe on any Ethereum-compatible wallet or hardware wallet. Please make sure that you can set up custom blockchain networks. Otherwise, you might not be able to see or send LYX on the LUKSO blockchain.
+Please ensure you can set custom blockchain networks within your blockchain wallet. Otherwise, you might not be able to see or send LYX within your wallet.
+
+:::
 
 ## Where can LYX be traded?
 
-At the current state, beginning of July 2023, there are no exchanges or services directly supporting LYX. However, as soon as exchanges migrate their token, there will be available trading. 3rd party services might even create decentralized marketplaces later on.
+You can officially trade LYX using the following platforms:
+
+- [KuCoin](https://www.kucoin.com/announcement/en-kucoin-has-completed-the-token-swap-of-lyxe-to-lyx-20230721)
+- [Gate.io](https://www.gate.io/article/33153)
+- [Ledger Live](https://support.ledger.com/hc/en-us/articles/15847276545053-Lukso-LYX-?docs=true).
```

Comment:
Im not sure Ledger Live has a trading feature. Maybe swap at least or onramp but I won't put that in the "trading" part.